src/formatters/stringFormatter.js

```diff
@@ -1,23 +1,16 @@
 import chalk from "chalk"
+import path from "path"
 import _ from "lodash"
-import formatter from "postcss-reporter/lib/formatter"
+import symbols from "log-symbols"
+import table, { getBorderCharacters } from "table"
+import utils from "postcss-reporter/lib/util"
 
-const minimalFormatter = formatter({
-  noIcon: true,
-  noPlugin: true,
-})
-
-export default function (results) {
-  let output = invalidOptionsFormatter(results)
-  output += deprecationsFormatter(results)
+const marginWidths = 9
 
-  return results.reduce((output, result) => {
-    output += minimalFormatter({
-      messages: result.warnings,
-      source: result.source,
-    })
-    return output
-  }, output)
+const levelColors = {
+  "info": "blue",
```

Comment:
Do we have a "info" severity? Is that something we [need to define](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/configuration.md#severities-error--warning). Or is the defined by the user?

Also, do we need to decide between ["advice"](https://github.com/stylelint/stylelint/issues/776) and "info"?